Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -663,6 +662,37 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         // no pay button tap because linked account is stubbed/fake in UI test
     }
 
+    func testGrabPayPaymentMethod() throws {
```

Comment:
Let's add a test to `PaymentSheetLPMConfirmFlowTests.swift`, should be super simple and we'll get full confirm flow coverage.